packages/p2p-v2/src/components/Input/Input.scss

```diff
@@ -0,0 +1,50 @@
+.p2p-v2-input {
+    margin: 0.5rem 0 3.6rem;
+
+    @include mobile {
+        margin-bottom: 1rem;
+    }
+
+    &__error {
+        padding-top: 0.2rem;
+        position: absolute;
+        margin-left: 1.6rem;
+
+        @include mobile {
+            box-decoration-break: clone;
+            padding-top: 0.4rem;
+            margin-bottom: 0.6rem;
+            position: relative;
+            width: 296px;
+        }
+    }
+
+    &__field {
+        display: flex;
+        width: 360px;
```

Comment:
wouldn't it be better to have 100% width for the input field and set the width style in the wrapper or the parent from which input component is called.

Reply:
i did try, but then it was overlapping with the parent div, but currently im just following what the design has said for this, cause we don't want to make it super stretched out as we change the window size

Reply:
ok i managed to change it around 👍